### PR TITLE
refactor all DB schemas and api routes

### PR DESCRIPTION
**Pull request by jasonLavigneDev**

### CODE REVIEW

File: pages/api/school/[id].js [hunk 1]
```diff
@@ -2,40 +2,47 @@ import dbConnect from "../../../lib/dbConnect";
...
-}
\ No newline at end of file
+  switch (method) {
+    case "GET":
+      try {
+        const id = req.query.id;
+        const school = await School.findById(id);
+
+        // a discuter peut etre le catch suffit
+        if (!school)
```
Comment (remi-boivin) [inline pages/api/school/[id].js]: remove all checks null please

File: pages/api/transport/[id].js [hunk 2]
```diff
@@ -9,17 +9,17 @@ export default async function handler(req, res) {
...
+        const id = req.query.id;
         const transport = await Transport.findById(id);
 
+        // a discuter peut etre le catch suffit
         if (!transport)
           return res
             .status(400)
             .json({ success: false, error: "Not match for this Id!" });
 
+        // A deplacer dans le Get Server Side Props
```
Comment (remi-boivin) [inline pages/api/transport/[id].js]: Please add todo keyword :)

**Review by remi-boivin — CHANGES_REQUESTED** [review]

File: pages/api/transport/[id].js (hunk 2 above)
Comment (jasonLavigneDev) [reply pages/api/transport/[id].js]: i ve delete this snippet from BackEnd and send itself to Front-End .

File: pages/api/school/[id].js (hunk 1 above)
Comment (jasonLavigneDev) [reply pages/api/school/[id].js]: ok :)

File: pages/api/transport/[id].js (hunk 2 above)
Comment (remi-boivin) [reply pages/api/transport/[id].js]: Okay even better :)

File: pages/api/school/[id].js (hunk 1 above)
Comment (remi-boivin) [reply pages/api/school/[id].js]: thanks

**Review by remi-boivin — APPROVED** [review]

**Pull request merged**